src/solc_0.8/common/Libraries/Address.sol

```diff
@@ -0,0 +1,189 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I think we could use the Openzepplin version of this file.
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Address.sol